configs/bigcommerce.json

```diff
@@ -1,37 +1,625 @@
 {
-  "index_name": "bigcommerce",
+  "index_name": "TESTING_INDEX",
   "start_urls": [
-    "https://support.bigcommerce.com/",
-    "https://support.bigcommerce.com/guides",
-    "https://support.bigcommerce.com/university",
-    "https://support.bigcommerce.com/documentation"
-  ],
+	{"url":"https://support.bigcommerce.com/articles/Learning/What-is-SEO","selectors_key": "guides"},
```

Comment:
I agree and I would prefer not to do it this way but based on the way the site is set up that craw config is not getting every URL in the site.